src/outfit.c

```diff
@@ -1450,31 +1450,31 @@ static void outfit_parseSBolt( Outfit* temp, const xmlNodePtr parent )
    /* Set short description. */
    temp->summary_raw = malloc( OUTFIT_SHORTDESC_MAX );
    l = 0;
-   SDESC_ADD(  l, temp, _("%s [%s]"), _(outfit_getType(temp)),
+   SDESC_ADD(  l, temp, "%s [%s]", _(outfit_getType(temp)),
          _(dtype_damageTypeToStr(temp->u.blt.dmg.type)) );
    dtype_raw( temp->u.blt.dmg.type, &dshield, &darmour, &dknockback );
-   SDESC_COLOURT(  l, temp, _("\n     %.0f%% vs armour"), 100., darmour*100. );
-   SDESC_COLOURT(  l, temp, _("\n     %.0f%% vs shield"), 100., dshield*100. );
-   SDESC_COND( l, temp, _("\n     %.0f%% knockback"), dknockback*100. );
-   SDESC_COND_COLOUR( l, temp, _("\n%.0f CPU"), temp->cpu );
-   SDESC_COND( l, temp, _("\n%.0f Mass"), temp->mass );
-   SDESC_ADD(  l, temp, _("\n%.0f%% Penetration"), temp->u.blt.dmg.penetration*100. );
-   SDESC_COND( l, temp, _("\n%.2f DPS [%.0f Damage]"),
-         1./temp->u.blt.delay * temp->u.blt.dmg.damage * (double)temp->u.blt.shots, temp->u.blt.dmg.damage * (double)temp->u.blt.shots );
-   SDESC_COND( l, temp, _("\n%.2f Disable/s [%.0f Disable]"),
-         1./temp->u.blt.delay * temp->u.blt.dmg.disable * (double)temp->u.blt.shots, temp->u.blt.dmg.disable * (double)temp->u.blt.shots );
-   SDESC_COND( l, temp, _("\n%.0f Hit Radius%s"), temp->u.blt.radius, (outfit_isProp(temp,OUTFIT_PROP_WEAP_FRIENDLYFIRE)?"!!":"") );
-   SDESC_ADD(  l, temp, _("\n%.1f Shots Per Second"), 1./temp->u.blt.delay );
-   SDESC_COND( l, temp, _("\n%.1f EPS [%.0f Energy]"),
-         1./temp->u.blt.delay * temp->u.blt.energy, temp->u.blt.energy );
-   SDESC_ADD(  l, temp, _("\n%s Range"), num2strU( temp->u.blt.range, 0 ) );
-   SDESC_ADD(  l, temp, _("\n%.0f Speed"), temp->u.blt.speed );
-   SDESC_COND( l, temp, _("\n%.1f second heat up"), temp->u.blt.heatup);
-   SDESC_COND( l, temp, _("\n%.1f Degree Dispersion"), temp->u.blt.dispersion*180./M_PI );
+   SDESC_COLOURT(  l, temp, "\n%s %.0f%%", _("Armour Damages:"), 100., darmour*100. );
+   SDESC_COLOURT(  l, temp, "\n%s %.0f%%", _("Shield Damages:"), 100., dshield*100. );
+   SDESC_COND( l, temp, "\n  %s %.0f%%", _("Knockback:"), dknockback*100. );
+   SDESC_COND_COLOUR( l, temp, "\n  %s %.0f", _("CPU:"), temp->cpu );
+   SDESC_COND( l, temp, "\n  %s %.0f", _("Mass:"), temp->mass );
+   SDESC_ADD(  l, temp, "\n  %s %.0f%%", _("Penetration:"), temp->u.blt.dmg.penetration*100. );
+   SDESC_COND( l, temp, "\n  %s %.2f [%.0f /sec]", _("Damages:"), temp->u.blt.dmg.damage * (double)temp->u.blt.shots, 
```

Comment:
"sec", "deg", "/sec", "min:", and "duration" in the formats should be translatable.

Reply:
:heavy_check_mark: